app/serializers.py

```diff
@@ -84,3 +84,31 @@ def update(self, instance, validated_data):
         instance.availability = validated_data.get('availability', instance.availability)
         instance.save()
         return instance
+
+
+class RequestSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Request
+        fields = [
+            'skill',
+            'description',
+            'mentee',
+            'mentor',
+            'status'
+        ]
+
+    def update(self, instance, validated_data):
+        skill = validated_data.get('skill')
+        if skill:
+            instance.skill.add(*skill)
+
+        description = validated_data.get('description')
```

Comment:
Why do we need to do this?

Reply:
Why can't we handle description the way we n\handled mentor, mentee and status?

Reply:
@Del-sama, I was thinking if we could add skill, why not add description too. Guess we don't want to do that.

I've updated to the way we handled the mentor, mentee and status anyways.